lib/sidekiq/api.rb

```diff
@@ -32,6 +32,7 @@ def dead_size
     end
 
     def enqueued
+      maybe_fetch_stats_slow!
```

Comment:
I don't think you need these explicit calls. Just make `stat` smarter?

Reply:
done in https://github.com/mperham/sidekiq/pull/4936/commits/a3028fc6e5d14179e0599e2001c92816e8ef985b